src/main/java/it/gov/pagopa/ranking/dto/mapper/OnboardingRankingRequest2RankingRequestsApiDTOMapper.java

```diff
@@ -0,0 +1,23 @@
+package it.gov.pagopa.ranking.dto.mapper;
+
+import it.gov.pagopa.ranking.dto.RankingRequestsApiDTO;
+import it.gov.pagopa.ranking.model.OnboardingRankingRequests;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OnboardingRankingRequest2RankingRequestsApiDTOMapper {
+
+    public RankingRequestsApiDTO apply(OnboardingRankingRequests onboardingRankingRequests, String organizationId) {
```

Comment:
organizationId should be alredy available inside onboardingRankingRequest, it's not necessary having it as second parameter